Makefile

```diff
@@ -67,7 +70,6 @@ bin:
 clean:
 	rm -rf bin/
 	rm -rf .gopath/
-	rm -rf src/atc_thrift/
```

Comment:
I am not sure it is a good idea to nuke atc_thrift all the time when running clean. The issue with deleting it means that we need to add the thrift compiler to the toolchain. Hardly anybody may need to fiddle with the thrift spec.